Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/StampToolIntegrationTest.java

```diff
@@ -114,6 +116,52 @@ public void testBorders() {
 		assertEquals(bottomRight, Color.BLACK);
 	}
 
+	@Test
+	public void testStampToolConsidersLayerOpacity() {
+		onToolBarView()
+				.performSelectTool(ToolType.SHAPE);
+
+		onShapeToolOptionsView()
+				.performSelectShape(DrawableShape.RECTANGLE);
+
+		onTopBarView()
+				.performClickCheckmark();
+
+		onToolBarView()
+				.performSelectTool(ToolType.STAMP);
+
+		LayerMenuViewInteraction.onLayerMenuView()
+				.performOpen()
+				.performSetOpacityTo(50, 0)
+				.performClose();
```

Comment:
Why are they formatted like this? Seems weid, as if the max line length was set too short

Reply:
This is default codestyle. We have it so you directly see what view it is and what actions are performed on it